### PR TITLE
fix: make multilevel forms work on IE10 and IE11 #4048

### LINKED ISSUE #4048: fix: make multilevel forms work on IE10 and IE11
# Bug Report

## User Story

As a donor on an older browser (Internet Explorer 10 and 11), I want multilevel forms to work when switching the amount.

## Current Behavior
On the latest version, the form does not work in switching the amount.

this error is in the console:
> On SCRIPT1014: Invalid character
> give.js (21,28016)

That appears to be highlighting a backtick ` character on line 49 of give.js

## Expected Behavior
I expect it to work on ie10 and ie 11

## Bug Type
- [x] This bug describes functionality that once worked as expected in version 2.3.1 (at least)

## Steps to Reproduce
1. Install Give on the latest version
2. Create a form that is multilevel with either radios or buttons
3. Load the form in IE 10 or IE 11
4. Try to change the amount on the form

## Related
https://secure.helpscout.net/conversation/802835825/36115/

## Acceptance Criteria
- [ ] the form works in IE10 and IE 11
- [ ] The fix does not affect newer browsers

## Environment
<details>
    <summary>WordPress System Info</summary>
### WordPress Environment ###

Home URL: https://livegive.wpsteward.com
Site URL: https://livegive.wpsteward.com
WP Version: 5.1.1
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
Show on Front: posts
Table Prefix Length: wp_8138bfdc07_
Table Prefix Length: 14
Table Prefix Status: Acceptable
Admin AJAX: Accessible
Registered Post Statuses: publish, future, draft, pending, private, trash, auto-draft, inherit, request-pending, request-confirmed, request-failed, request-completed, refunded, failed, revoked, cancelled, abandoned, processing, preapproval

### Server Environment ###

Hosting Provider: DBH: localhost, SRV: livegive.wpsteward.com
TLS Connection: Connection uses TLS 1.2
TLS Connection: Probably Okay
Server Info: Apache/2.4.38 (Unix) OpenSSL/1.0.1f
PHP Version: 7.3.3
PHP Post Max Size: 1 GB
PHP Time Limit: 30
PHP Max Input Vars: 5000
PHP Max Upload Size: 512 MB
cURL Version: ❌ 7.35.0, OpenSSL/1.0.1f - We recommend a minimum cURL version of 7.40.
SUHOSIN Installed: –
MySQL Version: ❌ 5.5.62 - We recommend a minimum MySQL version of 5.6. See: WordPress Requirements
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
gzip: ✔
GD Graphics Library: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Give Configuration ###

Give Version: 2.4.3
Give Cache: Enabled
Database Updates: All DB Updates Completed.
Database Tables: ✔ wp_8138bfdc07_give_donors -  1.0✔ wp_8138bfdc07_give_donormeta -  1.0✔ wp_8138bfdc07_give_comments -  1.0✔ wp_8138bfdc07_give_commentmeta -  1.0✔ wp_8138bfdc07_give_sessions -  1.0✔ wp_8138bfdc07_give_logs -  1.0✔ wp_8138bfdc07_give_logmeta -  1.0✔ wp_8138bfdc07_give_formmeta -  1.0✔ wp_8138bfdc07_give_sequential_ordering -  1.0✔ wp_8138bfdc07_give_donationmeta -  1.0
Give Cache: Enabled
Give Cache: ✔New Donation✔Donation Receipt❌New Offline Donation❌Offline Donation Instructions❌New User Registration❌User Registration Information❌Donor Note✔Email access
Upgraded From: 2.4.2
Test Mode: Enabled
Currency Code: USD
Currency Position: Before
Decimal Separator: .
Thousands Separator: ,
Success Page: https://livegive.wpsteward.com/donation-confirmation/
Failure Page: –
Donation History Page: https://livegive.wpsteward.com/donation-history/
Give Forms Slug: /donations/
Enabled Payment Gateways: Test Donation, PayPal Standard
Default Payment Gateway: Test Donation
PayPal IPN Verification: Disabled
PayPal IPN Notifications: IPN received for #1626 ( 7G231521RB710244C ) on 12/28/2018 at 19:40. Status N/A
Donor Email Access: Enabled

### Inactive Plugins ###

Ben's Helper Functions: by BenUNC –
Disable Gutenberg: by Jeff Starr – 1.6
Email Cop: by Ashfame – 0.1.1
Give - AmeriCloud Payments: by AmeriCloud Solutions, Inc. – 1.3.1
Give - Authorize.net Gateway: by GiveWP – 1.4.5
Give - Currency Switcher: by GiveWP – 1.3.1
Give - Database HealthCheck: by WordImpress – 0.0.2
Give - Email Reports: by WordImpress – 1.1.2
Give - Fee Recovery: by GiveWP – 1.7.4
Give - Form Field Manager: by GiveWP – 1.4.1
Give - Google Analytics Donation Tracking: by GiveWP – 1.2.2
Give - MailChimp: by GiveWP – 1.4.3
Give - Manual Donations: by GiveWP – 1.4.3
Give - Mollie Gateway: by GiveWP – 1.2.1
Give - PayPal Pro Gateway: by GiveWP – 1.2.1
Give - PDF Receipts: by GiveWP – 2.3.3
Give - Per Form Gateways: by GiveWP – 1.0.2
Give - Recurring Donations: by GiveWP – 1.8.5
Give - Recurring Donations: by GiveWP – 1.8.10
Give - Stripe Gateway: by GiveWP – 2.1.6
Give - Tributes: by GiveWP – 1.5.2
Give - Zapier: by WordImpress – 1.2.1
Google Analytics for WordPress by MonsterInsights: by MonsterInsights – 7.4.2
Hello Dolly: by Matt Mullenweg – 1.6
Loco Translate: by Tim Whitlock – 2.2.0
Query Monitor: by John Blackbourn – 3.3.2
Shortcode in Menus: by Gagan Deep Singh – 3.5
User Switching: by John Blackbourn & contributors – 1.4.2
WordPress Event Ticketing: by Tickera.com – 3.3.2
WP Crontrol: by John Blackbourn & contributors – 1.6.2

### Active MU Plugins ###

api-hacks.php: by  –
per-form-currency.php: by  –

### Theme ###

Name: Twenty Seventeen
Version: 2.1
Author URL: https://wordpress.org/
Child Theme: No – If you're modifying Give on a parent theme you didn't build personally, then we recommend using a child theme. See: How to Create a Child Theme
</details>

### PR DESCRIPTION
**Pull request by ravinderk**
## Description
This PR will resolve #4048 

## How Has This Been Tested?
https://www.useloom.com/share/9134c7b3c3b543f5ac8e9605b3b02ae2

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -4,13 +4,12 @@
       "@babel/preset-env",
       {
 		"targets": {
-		  "node": "current"
+		  "browsers": "last 2 version, > 1%, not dead, ie >= 10"
```
Comment (ravinderk) [inline .babelrc]: @DevinWalker @kevinwhoffman I added support for old browsers. Do you want to edit this?

Comment (ravinderk) [reply .babelrc]: update `> 1%` to `> 0.05%`

Comment (kevinwhoffman) [reply .babelrc]: @ravinderk Remove `not dead` as discussed on call.

Comment (ravinderk) [reply .babelrc]: https://github.com/impress-org/give/pull/4050/commits/e7ab84bd902b15ed93ac4ef91a4b41188a3568e1

**Pull request merged**